File: Ontolog-Module/Ontolog-Module/DataClasses/clsClasses.vb
Public Class clsClasses
    Private objOItem_Class_Root As clsOntologyItem
    Private objOItem_Class_Logstate As clsOntologyItem
    Private objOItem_Class_Directions As clsOntologyItem
    Private objOItem_Class_System As clsOntologyItem
    Private objOItem_Class_Ontologies As clsOntologyItem
    Private objOItem_Class_OntologyItems As clsOntologyItem
    Private objOItem_Class_OntologyRelationRule As clsOntologyItem
    Private objOItem_Class_OntologyJoin As clsOntologyItem
    Private objOItem_Class_Server As clsOntologyItem

    Private objTypes As New clsTypes

    Public ReadOnly Property OItem_Class_Server As clsOntologyItem
        Get
            Return objOItem_Class_Server
        End Get
    End Property

    Public ReadOnly Property OItem_Class_Root As clsOntologyItem
        Get
            Return objOItem_Class_Root
        End Get
    End Property

    Public ReadOnly Property OItem_Class_Directions As clsOntologyItem
        Get
            Return objOItem_Class_Directions
        End Get
    End Property

    Public ReadOnly Property OItem_Class_Logstate As clsOntologyItem
        Get
            Return objOItem_Class_Logstate
        End Get
    End Property

    Public ReadOnly Property OItem_Class_Ontologies As clsOntologyItem
        Get
            Return objOItem_Class_Ontologies
        End Get
    End Property

    Public ReadOnly Property OItem_Class_OntologyItems As clsOntologyItem
        Get
            Return objOItem_Class_OntologyItems
        End Get
    End Property

    Public ReadOnly Property OItem_Class_OntologyJoin As clsOntologyItem
        Get
            Return objOItem_Class_OntologyJoin
        End Get
    End Property

    Public ReadOnly Property OItem_Class_OntologyRelationRule As clsOntologyItem
        Get
            Return objOItem_Class_OntologyRelationRule
        End Get
    End Property

    Public ReadOnly Property OItem_Class_System As clsOntologyItem
        Get
            Return objOItem_Class_System
        End Get
    End Property
    Public Sub New()
        objOItem_Class_Root = New clsOntologyItem("49fdcd27e1054770941d7485dcad08c1", "Root", "dbbfc1a00c2e483684340a7b7a8b4b52")
        objOItem_Class_System = New clsOntologyItem("665dd88b792e4256a27a68ee1e10ece6", "System", objOItem_Class_Root.GUID, objTypes.ClassType)
        objOItem_Class_Logstate = New clsOntologyItem("1d9568afb6da49908f4d907dfdd30749", "Logstate", objOItem_Class_System.GUID, objTypes.ClassType)
        objOItem_Class_Directions = New clsOntologyItem("12de02469d84416faa45980efcb9db9b", "Directions", objOItem_Class_System.GUID, objTypes.ClassType)
        objOItem_Class_Ontologies = New clsOntologyItem("eb411e2ff93d4a5ebbbac0b5d7ec0197", "Ontologies", objOItem_Class_System.GUID, objTypes.ClassType)
        objOItem_Class_OntologyItems = New clsOntologyItem("d3f72a683f6146a48ff381db05997dc8", "Ontology-Items", objOItem_Class_Ontologies.GUID, objTypes.ClassType)
        objOItem_Class_OntologyRelationRule = New clsOntologyItem("925f489dec8d4130a418fcb022a4c731", "Ontology-Relation-Rule", objOItem_Class_Ontologies.GUID, objTypes.ClassType)
        objOItem_Class_OntologyJoin = New clsOntologyItem("aab30dd04faf4386896016218132b110", "Ontology-Join", objOItem_Class_Ontologies.GUID, objTypes.ClassType)
        objOItem_Class_Server = New clsOntologyItem("d7a03a35875142b48e0519fc7a77ee91", "Server", objOItem_Class_System.GUID, objTypes.ClassType)


    End Sub
End Class

File: Ontolog-Module/OntologyClasses/DataClasses/clsClasses.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using OntologyClasses.BaseClasses;

namespace OntologyClasses.DataClasses
{
    public class clsClasses
    {
        public clsOntologyItem OItem_Class_Root { get; private set; }
        public clsOntologyItem OItem_Class_Logstate { get; private set; }
        public clsOntologyItem OItem_Class_Directions { get; private set; }
        public clsOntologyItem OItem_Class_System { get; private set; }
        public clsOntologyItem OItem_Class_Ontologies { get; private set; }
        public clsOntologyItem OItem_Class_OntologyItems { get; private set; }
        public clsOntologyItem OItem_Class_OntologyRelationRule { get; private set; }
        public clsOntologyItem OItem_Class_OntologyJoin { get; private set; }
        public clsOntologyItem OItem_Class_Server { get; private set; }

        private clsTypes objTypes = new clsTypes();

        public clsClasses()
        {
            OItem_Class_Root = new clsOntologyItem {GUID = "49fdcd27e1054770941d7485dcad08c1", Name = "Root"};
            OItem_Class_System = new clsOntologyItem{ GUID = "665dd88b792e4256a27a68ee1e10ece6", Name = "System", GUID_Parent = OItem_Class_Root.GUID, Type = objTypes.ClassType};
            OItem_Class_Logstate = new clsOntologyItem { GUID = "1d9568afb6da49908f4d907dfdd30749", Name = "Logstate", GUID_Parent = OItem_Class_System.GUID, Type = objTypes.ClassType };
            OItem_Class_Directions = new clsOntologyItem { GUID = "12de02469d84416faa45980efcb9db9b", Name = "Directions", GUID_Parent = OItem_Class_System.GUID, Type = objTypes.ClassType };
            OItem_Class_Ontologies = new clsOntologyItem { GUID = "eb411e2ff93d4a5ebbbac0b5d7ec0197", Name = "Ontologies", GUID_Parent = OItem_Class_System.GUID, Type = objTypes.ClassType };
            OItem_Class_OntologyItems = new clsOntologyItem { GUID = "d3f72a683f6146a48ff381db05997dc8", Name = "Ontology-Items", GUID_Parent = OItem_Class_Ontologies.GUID, Type = objTypes.ClassType };
            OItem_Class_OntologyRelationRule = new clsOntologyItem { GUID = "925f489dec8d4130a418fcb022a4c731", Name = "Ontology-Relation-Rule", GUID_Parent = OItem_Class_Ontologies.GUID, Type = objTypes.ClassType };
            OItem_Class_OntologyJoin = new clsOntologyItem { GUID = "aab30dd04faf4386896016218132b110", Name = "Ontology-Join", GUID_Parent = OItem_Class_Ontologies.GUID, Type = objTypes.ClassType };
            OItem_Class_Server = new clsOntologyItem { GUID = "d7a03a35875142b48e0519fc7a77ee91", Name = "Server", GUID_Parent = OItem_Class_System.GUID, Type = objTypes.ClassType };
        }
    }
}
